test/e2e/kubernetes/job/job.go

```diff
@@ -57,6 +60,45 @@ func CreateJobFromFile(filename, name, namespace string) (*Job, error) {
 	return job, nil
 }
 
+// CreateWindowsJobFromTemplate will create a Job from file with a name
+func CreateWindowsJobFromTemplate(filename, name, namespace, servercore string) (*Job, error) {
```

Comment:
@jackfrancis - looks like you were the last to update the `validate-dns-windows.yaml` area. What do you think of this approach?